packages/core/content-type-builder/admin/src/components/AttributeIcon.tsx

```diff
@@ -64,14 +61,21 @@ const IconBox = styled(Box)`
   }
 `;
 
-const AttributeIcon = ({ type, customField, ...rest }) => {
+export type IconByType = keyof typeof iconByTypes;
+
+type AttributeIconProps = {
+  type: IconByType;
+  customField?: string | null;
+};
+
+export const AttributeIcon = ({ type, customField = null, ...rest }: AttributeIconProps) => {
   const customFieldsRegistry = useCustomFields();
 
-  let Compo = iconByTypes[type];
+  let Compo: any = iconByTypes[type];
 
   if (customField) {
-    const { icon } = customFieldsRegistry.get(customField);
-
+    const customFieldObject = customFieldsRegistry.get(customField);
```

Comment:
This is probably on the helper-plugin's side, but this customField type isn't valid